alacritty.yml

```diff
@@ -63,10 +63,20 @@ window:
   startup_mode: Windowed
 
   # Window title
-  #title: Alacritty
+  title: Alacritty
 
   # Window class (Linux only):
-  #class: Alacritty
+  class:
+    # Application instance name
+    name: Alacritty
+    # General application class
+    class: Alacritty
+
+  # Override GTK theme variant (Linux only).  "None" does not override default.
+  # Most themes support the following variants:
+  #   - light
+  #   - dark
```

Comment:
This should be formatted to match the other configuration options.

Something like this should be better:

```yml
# GTK theme variant (Linux only)
# 
# Override the variant of the GTK theme. Commonly supported values are `dark` and `light`.
# Set this to `None` to use the default theme variant.
```